client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/task-helper.ts

```diff
@@ -129,6 +132,19 @@ export function getEnhancedTasks(
 						},
 					};
 					break;
+				case 'unlock_styles':
+					if ( ! isEnabled( 'limit-global-styles' ) || ! globalStylesInUse ) {
```

Comment:
I think we should use `shouldLimitGlobalStyles` here instead of `isEnabled( 'limit-global-styles' )`, since `shouldLimitGlobalStyles` is a function of the feature check.

Reply:
> I think we should use `shouldLimitGlobalStyles` here

That would prevent the step from showing as completed.